### PR TITLE
Howto loki

### LINKED ISSUE #12587: docs: Metrics article mentions missing grafana board and the document says nothing about Loki datasource for the new board
*Please describe the question or issue you're facing with "How to monitor metrics - Canonical LXD documentation".*

Article describes LXD, Prometheus and Grafana configuration to have Grafana dashboard to display LXD cluster parameters.
1. Screenshot mentions dashboard 15726 which is no longer available
2. Text mentions dashboard 19131, which has been changed to have both Prometheus and Loki input sources, but the article says nothing about Loki data source. 
Loki datasource has been introduced to the dashboard in September 2023 and it is a required parameter in Grafana. So at the moment the article cannot be used to configure Grafana.

---
*Reported from: https://documentation.ubuntu.com/lxd/en/latest/metrics/*

### PR DESCRIPTION
**Pull request by escabo**
Added Loki howto including instructions from this request: https://discourse.ubuntu.com/t/how-to-set-labels-for-loki/42317/10
Also fixes #12587

### CODE REVIEW

**Comment by tomponline** [comment]
@simondeziel @escabo is a member of the LXD team, but its showing him as not passing the CLA check, any ideas?

**Comment by tomponline** [comment]
Doc checks failing with:

```
 .tmp/doc/howto/logs_loki.md:60: MD012 Multiple consecutive blank lines
.tmp/doc/howto/logs_loki.md:61: MD012 Multiple consecutive blank lines
```

**Review by tomponline — COMMENTED** [review]
Because the docs go into the snap, any practical reduction in image size would be appreciated.

**Comment by tomponline** [comment]
Please can you remove the merge commit `749c5a3db564cbf767f8a7025447cb32a7fbd07e` from this PR?

**Comment by simondeziel** [comment]
> Because the docs go into the snap, any practical reduction in image size would be appreciated.

I vaguely remember that those PNGs were minified prior to adding them. Could you do the same with those new/updated ones, please?

File: doc/howto/grafana.md
Comment (ru-fu) [inline doc/howto/grafana.md:32]:
```suggestion
   1. In the {guilabel}`URL` field, enter the address of your Prometheus installation (`http://localhost:9090/`).
```

Comment (ru-fu) [reply doc/howto/grafana.md:32]: With or without the slash at the end? The screenshot has it without.

File: doc/howto/grafana.md
Comment (ru-fu) [inline doc/howto/grafana.md:44]:
```suggestion
   1. In the {guilabel}`URL` field, enter the address of your Loki installation (`http://localhost:3100/`).
```

Comment (ru-fu) [reply doc/howto/grafana.md:44]: Same question about the slash.

File: doc/howto/grafana.md
Comment (ru-fu) [inline doc/howto/grafana.md:52]:
```suggestion
   1. Go back to the Basic (quick setup) panel and now choose {guilabel}`Dashboards` > {guilabel}`Import a dashboard`.
```

File: doc/howto/logs_loki.md
Comment (ru-fu) [inline doc/howto/logs_loki.md:51]:
```suggestion
## Add labels
```

File: doc/metrics.md
Comment (ru-fu) [inline doc/metrics.md:1]: Can you add a link to the Grafana how-to in the intro?

```
<!-- Include start metrics intro -->
LXD collects metrics for all running instances as well as some internal metrics.
These metrics cover the CPU, memory, network, disk and process usage.
They are meant to be consumed by Prometheus, and you can use Grafana to display the metrics as graphs.
See {ref}`provided-metrics` for lists of available metrics and {ref}`grafana` for instructions on how to display the metrics in Grafana.
<!-- Include end metrics intro -->
```
(this is from line 12-16, only the last part of the last sentence is new)

File: doc/howto/logs_loki.md
Comment (ru-fu) [reply doc/howto/logs_loki.md:11]: Add a link to the Grafana how-to here:

```
You can then display the log events in Grafana. See {ref}`grafana` for instructions.
```

File: doc/howto/grafana.md
Comment (ru-fu) [inline doc/howto/grafana.md:26]:
```suggestion
   1. From the Basic (quick setup) panel, choose {guilabel}`Data Sources`.

      ![Choose data source in Grafana](/images/grafana_welcome.png)
```

**Review by ru-fu — COMMENTED** [review]
Thanks, this looks great!
Some comments for consistency.

**Comment by tomponline** [comment]
@escabo as @ru-fu has provided some feedback, I thought it would be prudent to let you know that because we dont use squash merges in LXD, the full commit list of your PR will land in the main branch of LXD. As such we don't want to see additional commits to show the fixes @ru-fu has suggested, but instead the changes should be squashed or fixed up into the existing commit(s).

As you've only got a single commit here, then `git commit --amend` could be useful.

**Review by tomponline — DISMISSED** [review]

**Review by tomponline — APPROVED** [review]

**Comment by tomponline** [comment]
I'll just wait for @ru-fu approval on the content

**Review by ru-fu — APPROVED** [review]
Thanks! :1st_place_medal:

**Pull request merged**
